lib/contracts.js

```diff
@@ -10,6 +10,10 @@ const {
 
 export const runUpdateContractsLoop = async ({ provider, abi, contracts }) => {
   while (true) {
```

Comment:
Should we add a small delay (e.g. 2 seconds) before we start the loop to avoid DDoSing w3name in case Station Core is frequently restarted? For example, if somebody is operating Station Core and has a mechanism to restart Core immediately after it exits.

```suggestion
export const runUpdateContractsLoop = async ({ provider, abi, contracts }) => {
  await timers.setTimeout(2_000)
  while (true) {
```

Reply:
great idea! 👏